Neos.Neos/Documentation/ExtendingNeos/UiExtensibility.rst

```diff
@@ -0,0 +1,371 @@
+.. _ui-extensibility:
+
+=====================================
+Neos User Interface Extensibility API
+=====================================
+
+At the heart of the Neos UI lies the system of registries – key-value stores that contain system components. The registries are populated through the `manifest` API command that is exposed through the neos-ui-extensibility package.
+
+Inspector-specific Registries
+=============================
+
+Editors
+-------
+
+Way to retrieve: `globalRegistry.get('inspector').get('editors')`
+
+Contains all inspector editors.
+
+The key is an editor name (such as Neos.Neos/Inspector/Editors/SelectBoxEditor), and the values
+are objects of the following form:
+
+.. code-block:: javascript
+
+  {
+    component: TextInput // the React editor component to use. Required
+    hasOwnLabel: true|false // does the component render the label internally, or not?
+  }
+
+Component Wiring
+~~~~~~~~~~~~~~~~
+
+Every component gets the following properties (see EditorEnvelope/index.js)
+
+- identifier: an identifier which can be used for HTML ID generation
```

Comment:
Either start in uppercase or lowercase after the colon, but the same way for each item